sw/device/silicon_creator/imm_rom_ext/prebuilts/README.md

```diff
@@ -0,0 +1,4 @@
+## NOP IM_ROM_EXT
+
+* The `nop.o` is built by the following command:
```

Comment:
Perhaps you could also elaborate on the purpose of this "NOP immutable ROM_EXT"? I am assuming this is what would be used for testing?

Reply:
Done. I sightly updated the markdown file.